examples/validating-public-input/contracts/validate_batch_inclusion.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+# cd to the directory of this script so that this can be run from anywhere
+parent_path=$( cd "$(dirname "${BASH_SOURCE[0]}")" || exit 1 ; pwd -P )
+
+# At this point we are in aligned-example
+cd "$parent_path" || exit 1
+
+# check if the number of arguments is correct
+if [ "$#" -ne 3 ]; then
+    echo "Usage: verify_batch_inclusion.sh <FIBONACCI_VALIDATOR_ADDRESS> <DATA_FILE_NAME> <VERIFIER_ID>"
+    exit 1
+fi
+
+FIBONACCI_VALIDATOR_ADDRESS=$1
+DATA_FILE_NAME=$2
+VERIFIER_ID=$3
+
+if [ -z "$RPC_URL" ]; then
+    echo "RPC_URL is not set. Please set it in .env"
+    exit 1
+fi
+
+if [ -z "$PRIVATE_KEY" ]; then
+    echo "PRIVATE_KEY is not set. Please set it in .env"
+    exit 1
+fi
+
+proof_commitment=$(jq -r '.proof_commitment' "../aligned-integration/batch_inclusion_data/$DATA_FILE_NAME.json")
+pub_input_commitment=$(jq -r '.pub_input_commitment' "../aligned-integration/batch_inclusion_data/$DATA_FILE_NAME.json")
+program_id_commitment=$(jq -r '.program_id_commitment' "../aligned-integration/batch_inclusion_data/$DATA_FILE_NAME.json")
+proof_generator_addr=$(jq -r '.proof_generator_addr' "../aligned-integration/batch_inclusion_data/$DATA_FILE_NAME.json")
+batch_merkle_root=$(jq -r '.batch_merkle_root' "../aligned-integration/batch_inclusion_data/$DATA_FILE_NAME.json")
+merkle_proof=$(jq -r '.merkle_proof' "../aligned-integration/batch_inclusion_data/$DATA_FILE_NAME.json")
+verification_data_batch_index=$(jq -r '.verification_data_batch_index' "../aligned-integration/batch_inclusion_data/$DATA_FILE_NAME.json")
+pub_input=$(jq -r '.pub_input' "../aligned-integration/batch_inclusion_data/$DATA_FILE_NAME.json")
```

Comment:
I would remove the `.json` to let the user put the full file name

Reply:
Fixed here: [20b8aae](https://github.com/yetanotherco/aligned_layer/pull/1195/commits/20b8aae8b3512f3225b119c142cfb7840e284a1f)!